src/app/student/events/page.tsx

```diff
@@ -0,0 +1,19 @@
+import { EventsTimeline } from "@/app/student/_components/EventsTimeLine"
+import { Page } from "@/components/shared/Page"
+import { fetchEvents } from "@/components/shared/hooks/api/useEvents"
+import { Suspense } from "react"
+
+export default async function StudentEventPage() {
+	const events = await fetchEvents()
+
+	return (
+		<Page.Background withIndents>
+			<Page.Boundary>
+				<Page.Header>Events</Page.Header>
+				<Suspense>
```

Comment:
If I'm not mistaken suspense won't have any effect here since all the data is passed as props, if there would have been a query within this component suspense would make sense